functions/supplier/sftp-supplier/src/test/java/org/springframework/cloud/fn/supplier/sftp/SftpSupplierApplicationTests.java

```diff
@@ -399,8 +444,8 @@ private void deleteNewSource(Path newSource) throws IOException {
 	}
 
 	@SpringBootApplication
+	@Import(SpelExpressionConverterConfiguration.class)
 	static class SftpSupplierTestApplication {
 
 	}
-
```

Comment:
As I said: every class member has to be surrounded with blank lines, even before the last `}` in the class.